subsystems/SimpleMotorBase.java

```diff
@@ -6,23 +6,25 @@
 
 public abstract class SimpleMotorBase extends SubsystemBase {
     protected SpeedControllerGroup motorGroup;
-    protected double throttle;
+    protected double forwardThrottle;
+    protected double backwardThrottle;
 
-    public SimpleMotorBase(SpeedController[] motors, double throttle) {
+    public SimpleMotorBase(SpeedController motors, double forwardThrottle, double backwardThrottle) {
```

Comment:
it may be better to name this `motor` because we don't know whether it represents more than one